pkg/transforms/policyreport.go

```diff
@@ -90,7 +96,13 @@ func PolicyReportResourceBuilder(pr *PolicyReport) *PolicyReportResource {
 	for k := range categoryMap {
 		categories = append(categories, k)
 	}
-	node.Properties["rules"] = policies
+
+	policyList := policies.UnsortedList()
+	sort.Strings(policyList)
+
+	// "rules" is incorrect since there is a "rule" field in the results, but this is kept for backwards compatibility
+	node.Properties["rules"] = policyList
+	node.Properties["policies"] = policyList
```

Comment:
I don't want to duplicate this data, but we don't have any way to deprecate and remove the incorrect "rules" property name. 🤔

Reply:
For now, I just removed `policies` and commented in other parts of the code explaining the mismatch.